zerver/views/realm.py

```diff
@@ -599,6 +599,7 @@ def update_realm_user_settings_defaults(
         Annotated[int, check_int_in_validator(UserProfile.WEB_CHANNEL_DEFAULT_VIEW_CHOICES)]
     ]
     | None = None,
+    web_left_sidebar_unreads_count_summary: Json[bool] | None = None,
```

Comment:
Can you add a commit on top of this one sorting these parameters alphabetically, here and in the other views file? I think it'll make our life easier to not have the order be basically random.

Reply:
Yeah, a separate PR sounds good.